implementations/rust/ockam/ockam_api/src/nodes/service.rs

```diff
@@ -237,6 +250,11 @@ impl NodeManager {
             authorities: None,
             authenticated_storage,
             registry: Default::default(),
+            medic: {
+                let ctx = ctx.async_try_clone().await?;
+                tokio::spawn(medic.start(ctx))
```

Comment:
We ideally want to limit direct usage of tokio to `ockam_node` crate. It's totally fine for now though, just FYI